third-party/genuine-fantasy-press/forgotten-secrets/items.xml

```diff
@@ -0,0 +1,1324 @@
+		</description>
+		<setters>
+			<set name="category">Wondrous Items</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="0">0 lb.</set>
+			<set name="type">Wondrous Item</set>
+			<set name="rarity">Very Rare</set>
+			<set name="attunement">true</set>
+		</setters>
+	</element>
+
+	<element name="Miserable Statuette" type="Magic Item" source="The Compendium of Forgotten Secrets" id="ID_GFP_COFSA_MAGIC_ITEM_WONDROUS_ITEM_MISERABLE_STATUETTE">
+		<description>
+			<p>This small statue depicts a blank humanoid figure. As an action while holding it, you can touch a humanoid creature within reach. The target must make a DC 15 Wisdom saving throw. If they fail, they are forcefully sucked into the statuette and imprisoned, causing the face of the humanoid figure to change to match the captive. The target is immune to damage and effects while inside the statuette. You can expel the target from the statuette as an action. If you die or are no longer attuned to the statuette, the creature is instantly freed.</p>
+		</description>
+		<setters>
+			<set name="category">Wondrous Items</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="0">0 lb.</set>
+			<set name="type">Wondrous Item</set>
+			<set name="rarity">Very Rare</set>
+			<set name="attunement">true</set>
+		</setters>
+	</element>
+
+	<element name="Slave Collar" type="Magic Item" source="The Compendium of Forgotten Secrets" id="ID_GFP_COFSA_MAGIC_ITEM_WONDROUS_ITEM_SLAVE_COLLAR">
+		<description>
+			<p>This golden serpentine collar can’t be removed without a DC 18 Strength check. Failing this check inflicts a level of exhaustion on the wearer as it strangles them. You can place it on a restrained or incapacitated creature as an action. While worn, the wearer has disadvantage on attack rolls.</p>
+		</description>
+		<setters>
+			<set name="category">Wondrous Items</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="0">0 lb.</set>
+			<set name="type">Wondrous Item</set>
+			<set name="rarity">Common</set>
+			<set name="attunement">true</set>
+		</setters>
+	</element>
+
+	<element name="Claws of the Stalker" type="Magic Item" source="The Compendium of Forgotten Secrets" id="ID_GFP_COFSA_MAGIC_ITEM_WONDROUS_ITEM_CLAWS_OF_THE_STALKER">
+		<description>
+			<p>These gloves are equipped with retractable diamond claws at the fingertips. While wearing them, your unarmed strikes are considered weapons with the finesse property that deal 1d6 slashing damage, and you gain a climb speed equal to your walking speed.</p>
+		</description>
+		<setters>
+			<set name="category">Wondrous Items</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="0">0 lb.</set>
+			<set name="type">Wondrous Item</set>
+			<set name="rarity">Very Rare</set>
+			<set name="attunement">true</set>
+			<set name="slot">arms</set>
+		</setters>
+		<rules>
+			<stat name="speed:climb:misc" value="speed" />
```

Comment:
change to: `<stat name="speed:climb" value="speed" bonus="base" />` to apply a base climb speed